spec/models/form526_submission_spec.rb

```diff
@@ -46,12 +43,86 @@
         )
       end
 
+      let(:hypertension_form_json) do
```

Comment:
I don't quite understand this change, but I actually agree with the old format, because the `subject` isn't really a newly created form.

It's fine not to have a subject for this sort of very imperative-style method.

Reply:
This change was to force the context `the submission is for hypertension` to use the `form_for_hypertension` in the subject. Turns out that  just declaring the form in the line `let(:form_for_hypertension) do` isnt actually using in the context. So the form in the subject was still the one declared in the original subject, which had no hypertension disability.

Reply:
Gotcha, thanks for confirming that this fixes an existing bug in the rspec setup.

I think the setup can be made even clearer by DRYing up the repetition in the subjects. Since only the `form_json:` changes, you could have the HTN context override just the `let(:form_json)` (and similarly with tinnitus later on). That would read better too, as the context description would match the one override.